osx/step02_omero.sh

```diff
@@ -7,7 +7,7 @@ set -x
 
 export PATH=/usr/local/bin:$PATH
 export OMERO_DATA_DIR=${OMERO_DATA_DIR:-~/OMERO.data}
-export PSQL_SCRIPT_NAME=${PSQL_SCRIPT_NAME:-OMERO.sql}
+export PSQL_SCRIPT_NAME=${PSQL_SCRIPT_NAME:-~/OMERO.sql}
```

Comment:
That was the original idea of use the current directory (under `/usr/local`) as it was getting wiped out by the cleanup step. I can use `/tmp/OMERO.sql` if that makes sense.